webrender/src/prim_store.rs

```diff
@@ -504,21 +504,17 @@ pub struct TextShadowPrimitiveCpu {
 
 #[derive(Debug, Clone)]
 pub struct TextRunPrimitiveCpu {
-    pub font_key: FontKey,
+    pub font: FontInstance,
```

Comment:
This is the bit I was mentioning above that could just store the FontInstanceKey here.